server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -983,10 +992,17 @@ private Engine.IndexResult applyIndexOperation(
             verifyNotClosed(e);
             return new Engine.IndexResult(e, version, opPrimaryTerm, seqNo, sourceToParse.id());
         }
-
         return index(engine, operation);
     }
 
+    public void fieldInfos(FieldInfos fieldInfos) {
```

Comment:
nit: call it setFIeldInfos? And the other perhaps getFieldInfos?

Reply:
Done!